README.md

```diff
@@ -22,6 +22,11 @@ To display some data to the terminal just call:
 
     qrcode.generate('This will be a QRCode, eh!');
 
+You can even specify the error level (default is 'L'):
+    
+    qrcode.setErrorLevel(error='Q');
```

Comment:
You can do either or. I just felt that the documentation should be explicit about the parameter. I'm more than happy to change it if it fits your style better.